### ISSUE TITLE
Text input disappearing

### ISSUE THREAD

**Issue by LeslieDesser** [osTicket/osTicket#5556]
### Prerequisites
* [No ] Did you check the [osTicket forums](https://forum.osticket.com/)?
* [Yes] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

### Version 1.14.2

I pasted an Access SQL statement into a ticket reply and sent it.  On arrival it did not work.

On examination it turned out that the statement contained several 'not-equal' symbols which in Access are represented by a less-than and a greater-than symbol side by side (as here: <>).

osTicket eats up such pairs - they disappear when the input is saved.

Further testing on editing an FAQ I got the same result.

It must be that when the text is processed the pair of characters is treated as an empty HTML tag and removed.

Running on CentOS 7 php 7.2 or 7.3

PS:
I tried entering a reverse slash before each character. After the save all there was left was: \

Interestingly entering 'rev.slash less-than rev.slash greater-than appeared on this  github site as <> after save and the slashes were still there if I switched to Edit mode.

**Comment by JediKev** [comment]
@LeslieDesser 

Yes, this is mentioned in many other threads on the Forum (and I think here). It’s a combination of HTMLawed and other sanitization/balance methods we use stripping out code that is not whitelisted or what it deems "unbalanced" (security reasons). I believe that since you had `<...>` it deemed it an "unbalanced" tag and stripped it. You can modify the code to not strip content (this is not recommended) or you could look into whitelisting certain code in HTMLawed and the other sanitization methods (`include/class.format.php`).

Cheers.

**Comment by jianfzhu** [comment]
@JediKev 
can you show me how to modify the file to allow user input <....>, thanks!
we meet the same problem.

**Comment by antiuser** [comment]
@JediKev same problem....
We are using FAQ for some configuration of linux applications (such as apache). And osTicket text editor modify some characters (such as <>)

**Comment by nobe80** [comment]
Same problem here! It is very annoying every time we have to insert logfiles snippets for example.
So the <..> characters has to be allowed for the staff member of osticket. 
Why there is no such settings in there? For incoming input ok i understand the security risk 
but not form insinde to outside. There is has to be allowed

**Comment by JediKev** [comment]
@nobe80 

It doesn't matter if you are an Agent, if someone somehow gets access to an Agent profile (possibly through another vuln, etc.) then it can be exploited.

We hope to have a better text editor, to allow things such as code snippets, etc. and to have better sanitization methods that will allow such characters in or after [v2.0](https://github.com/osTicket/roadmap) (full codebase rewrite). Stay tuned!

Cheers.
